kong/db/declarative/init.lua

```diff
@@ -136,4 +175,97 @@ function declarative.load_into_db(dc_table)
 end
 
 
+local function remove_nulls(tbl)
```

Comment:
I considered it, but:

* it is used only once in Kong proper (the other use is in the testsuite)
* I don't want people to think that this is the normal way of getting rid of `ngx.null` in responses (it is `opts.nulls`). The reason why we do this here is because this is a special situation (I didn't want to mess with the invariant in the schema library and DAO that `"insert"` mode always produces nulls).
* I'm trying not to fill `kong.tools.utils` with more things since it is already the bag of random stuff :)